UnifiedApi/Worker/Grid/TaskSubmitterWorkerService.cs

```diff
@@ -0,0 +1,387 @@
+///   https://github.com/aneoconsulting/ArmoniK.Samples/tree/main/Samples/SymphonyLike
+///   Samples.ArmoniK.Sample.SymphonyPackages
+/// </summary>
+[PublicAPI]
+[MarkDownDoc]
+public class TaskSubmitterWorkerService : ITaskSubmitterWorkerServiceConfiguration
+{
+  /// <summary>
+  /// </summary>
+  public TaskSubmitterWorkerService()
+  {
+    Configuration = GridWorkerExt.GetDefaultConfiguration();
+    Logger = GridWorkerExt.GetDefaultLoggerFactory(Configuration)
+                          .CreateLogger(GetType()
+                                          .Name);
+  }
+
+  /// <summary>
+  /// </summary>
+  /// <param name="loggerFactory"></param>
+  public TaskSubmitterWorkerService(ILoggerFactory loggerFactory)
+  {
+    LoggerFactory = loggerFactory;
+
+    Logger = loggerFactory.CreateLogger(GetType()
+                                          .Name);
+  }
+
+  /// <summary>
+  ///   ginScope
+  ///   Get access to Logger with Logger.Lo.
+  /// </summary>
+  public ILogger Logger { get; set; }
+
+  /// <summary>
+  ///   Get or Set SubSessionId object stored during the call of SubmitTask, SubmitSubTask,
+  ///   SubmitSubTaskWithDependencies or WaitForCompletion, WaitForSubTaskCompletion or GetResults
+  /// </summary>
+  public Session SessionId { get; set; }
+
+  /// <summary>
+  ///   Property to retrieve the sessionService previously created
+  /// </summary>
+  internal SessionPollingService SessionService { get; set; }
+
+  /// <summary>
+  ///   Map between ids of task and their results id after task submission
+  /// </summary>
+  public Dictionary<string, string> TaskId2OutputId
+    => SessionService.TaskId2OutputId;
+
+  //internal ITaskHandler TaskHandler { get; set; }
+
+  /// <summary>
+  ///   Return TaskOption coming from Client side
+  /// </summary>
+  public TaskOptions TaskOptions { get; set; } = new();
+
+  /// <summary>
+  ///   Get or Set Configuration
+  /// </summary>
+  public IConfiguration Configuration { get; set; }
+
+  /// <summary>
+  ///   The logger factory to create new Logger in sub class caller
+  /// </summary>
+  public ILoggerFactory LoggerFactory { get; set; }
+
+  public TaskContext TaskContext { get; set; }
+
+  /// <summary>
+  ///   Get or set the taskId (ONLY INTERNAL USED)
+  /// </summary>
+  public TaskId TaskId { get; set; }
+
+  /// <summary>
+  ///   The configure method is an internal call to prepare the ServiceContainer.
+  ///   Its holds several configuration coming from the Client call
+  /// </summary>
+  /// <param name="configuration">The appSettings.json configuration prepared during the deployment</param>
+  /// <param name="clientOptions">All data coming from Client within TaskOptions </param>
+  public void Configure(IConfiguration configuration,
+                        TaskOptions    clientOptions)
+  {
+    Configuration = configuration;
+
+    //Append or overwrite Dictionary Options in TaskOptions with one coming from client
+    TaskOptions.MergeFrom(clientOptions);
```

Comment:
that's a bug ! https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/pull/96
rebase ?

Reply:
ok I followed : [UnifiedApi/Worker/Grid/BaseService.cs](https://github.com/aneoconsulting/ArmoniK.Extensions.Csharp/commit/6c6d13d81c9812b5265eaf6cbc289ddf80628265#diff-66e13844479a78bc1af0483f428647f43e75ec94fe9c556cc73ef172d8c2ef59)
but it looks like to be only a set nomore a merge